src/main/java/games/strategy/triplea/ui/PoliticsPanel.java

```diff
@@ -119,7 +119,6 @@ public PoliticalActionAttachment waitForPoliticalAction(final boolean firstRun,
 
     final JDialog politicalChoiceDialog = new JDialog(m_parent, "Political Actions", true);
     final Insets insets = new Insets(1, 1, 1, 1);
-    int row = 0;
```

Comment:
There is a fixed number of rows in this panel.  Thus a variable row counter is not necessary.  I simply replaced the value of `row` with appropriate literals where it was used.